CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+## Technical Steering Committee (TSC)
+
+The Technical Steering Committee (the “TSC”) is responsible for all technical oversight of the OSS Review Toolkit project. 
+
+TSC members:
+
+* Frank Viernau ([@fviernau](https://github.com/fviernau))
+* Martin Nonnenmacher ([@mnonnenmacher](https://github.com/mnonnenmacher))
+* Sebastian Schuberth ([@sschuberth](https://github.com/sschuberth))
+* Thomas Steenbergen ([@tsteenbe](https://github.com/tsteenbe)) 
+
+TSC has elected not to have a TSC chair, but Thomas Steenbergen as spokesperson / project manager.
```

Comment:
- "The TSC"
- Instead of "elected" I'd say "chosen" or "decided" because the point exactly is that no one was "elected"
- Instead of "project manager" I'd prefer "representative" or something similar.